zhaquirks/tuya/ts0601_valve.py

```diff
@@ -0,0 +1,154 @@
+"""Tuya Garden Watering."""
+
+import logging
+from typing import Dict
+
+from zigpy.profiles import zha
+from zigpy.quirks import CustomDevice
+import zigpy.types as t
+from zigpy.zcl.clusters.general import Basic, Groups, Ota, Scenes, Time
+from zigpy.zcl.clusters.smartenergy import Metering
+
+from zhaquirks import DoublingPowerConfigurationCluster
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+from zhaquirks.tuya import TuyaLocalCluster
+from zhaquirks.tuya.mcu import (
+    DPToAttributeMapping,
+    TuyaAttributesCluster,
+    TuyaDPType,
+    TuyaMCUCluster,
+    TuyaOnOff,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class TuyaGardenWateringTimer(TuyaAttributesCluster):
```

Comment:
Where you've find those values like `cluster_id` and `attributes`? It is just made up?

Reply:
> Code in this PR looks suspiciously similar #1450

@rforro If you follow the discussion in the issue, I based my code on #1446 that was closed.
I didn't see the #1450.

But for both, I get deeper with the measurement instead of a flow (for a reported volume), check the issue #1556 for the follow up by @javicalle

Reply:
And I did the unit test :)

Reply:
Done as discussed in issue